modules/social_features/social_follow_content/social_follow_content.install

```diff
@@ -18,6 +18,29 @@ function social_follow_content_install() {
   _social_follow_content_set_permissions();
 }
 
+/**
+ * Set permissions and delete old functionality.
+ */
+function social_follow_content_update_8001() {
+  _social_follow_content_set_permissions();
+
+  $block = \Drupal::entityTypeManager()->getStorage('block')
+    ->load('exposedformfollowingfollowing');
+
+  if ($block) {
+    $block->delete();
+  }
+
+  $config_names = [
+    'views.view.following',
+    'block.block.exposedformfollowingfollowing',
```

Comment:
Is this not a duplicate of the `$block->delete()` above?

Reply:
Yes, it does the same